.pre-commit-config.yaml

```diff
@@ -41,7 +41,7 @@ repos:
           - flake8-simplify
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.910
+    rev: v0.910-1
```

Comment:
Kind of weird but it does seem to exist

Reply:
I just ran `pre-commit autoupdate` :shrug: